src/include/sof/audio/sof_dsm.h

```diff
@@ -0,0 +1,104 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later
+ *  Copyright (c) 2020 Maxim Integrated
+ */
+
+#ifndef __SOF_AUDIO_DSM_H__
+#define __SOF_AUDIO_DSM_H__
+
+#include <sof/platform.h>
+#include <sof/audio/component.h>
+
+#define MAX_CHANNELS 2
```

Comment:
Too generic, will probably collide with other private macros. Btw ditto on the MAXIM_ prefix if it's a maxim specific file (file will need renamed too if not generic.)

Reply:
Agreed. Shall fix this.